### ISSUE TITLE
upgrade to Grafana v8.3.1 regression : Can’t get subscriptions list or workspaces list in variable template for azure log analytics

### ISSUE THREAD

**Issue by ismzer77** [grafana/grafana#43106]
**What happened**:
workspaces() and subscriptions() function dosen't work after grafana upgrade to 8.3.1
**What you expected to happen**:
no regression, workspaces() should be list all workspaces 
no issue found before grafana upgrade, test OK with grafana 8.1.5
**How to reproduce it (as minimally and precisely as possible)**:
upgrade grafana to 8.3.1 (from 8.1.5 for example)
i use the same azure monitor pluging (0.3.1)
**Anything else we need to know?**:

**Environment**:
- Grafana version: 8.3.1
- Data source type & version: azure monitor 
- OS Grafana is installed on: ubunto
- User OS & Browser: linux/chrome
- Grafana plugins:azure monitor (0.3.1)
- Others:
![grafana_error](https://user-images.githubusercontent.com/69198797/146025890-232e8e24-96f3-40e1-ba69-19dae15a7e39.jpg)

**Comment by sarahzinger** [comment]
Hi @ismzer77 unfortunately I'm having trouble recreating this bug. I just checked out 8.1.5 and created 2 template variables, `workspaces()` and `subscriptions()` and they seem to load for me. But we did work on those helper functions around that time, so it seems very possible a bug was introduced, just having difficulty reproducing.

Would it be possible to send additional screenshots or additional information? Some things that could be helpful:
- if you have a json dashboard to share that would be great
- what pages you are seeing these errors on (is it the dashboard page? the edit panel page? etc)
- a screenshot of what your variables look like on the variables page
- any information on how you are loading the dashboards (are you importing them, creating them manually, etc)
- does it happen every time you load the dashboard or only sometimes?
